media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -5929,6 +5929,7 @@ static uint32_t DdiMedia_GetDrmFormatOfCompositeObject(uint32_t fourcc)
     case VA_FOURCC_Y800:
         return DRM_FORMAT_R8;
     case VA_FOURCC_P010:
+	return DRM_FORMAT_P010;
```

Comment:
Could you please replace the Tab key with space? Thanks.

Reply:
Thanks, updated.